modules/ROOT/pages/plugins/opensource/table.adoc

```diff
@@ -111,7 +111,7 @@ tinymce.init({
 });
 ----
 
```

Comment:
In the above, from the `Examples of various table setups` the heading level is incorrect as it's saying this is an option in the ToC. We need to remove an `=` from each heading. For example:
```
== Examples of various table setups

Here are some examples of configuration for common setups.

=== No default attributes or styles on tables

[source, js]
----
tinymce.init({
  selector: 'textarea',  // change this value according to your HTML
  plugins: 'table',
  table_default_attributes: {},
  table_default_styles: {}
});
----

=== Pixel based resizing

[source, js]
----
tinymce.init({
  selector: 'textarea',  // change this value according to your HTML
  plugins: 'table',
  table_sizing_mode: 'fixed'
});
----

=== Percentage based resizing

[source, js]
----
tinymce.init({
  selector: 'textarea',  // change this value according to your HTML
  plugins: 'table',
  table_sizing_mode: 'relative'
});
----
```

Reply:
Done